debian/control

```diff
@@ -0,0 +1,38 @@
+Source: rocksdb
+Maintainer: G. B. Versiani <guibv@yahoo.com>
+Priority: optional
+Standards-Version: 3.9.5
+Build-Depends: debhelper (>> 9),
+ devscripts,
+ hardening-includes
+Section: libs
+Homepage: http://rocksdb.org
+
+Package: librocksdb4.4-dev
```

Comment:
Do we have to hardcode this as 4.4? Is it possible to have a script that updates the version number to the current version?